Header header
int32[] map_size
float32[] current_viechle_pose
float32   current_viechle_yaw
float32[] current_viechle_vel
float32[] current_people_map
float32[] current_obs_map
float32[] current_parameters

int32     planning_success
int32     collision
int32     need_reset

float32   tracking_error
float32   reward

